src/main/java/com/cst438/controller/EnrollmentController.java

```diff
@@ -9,21 +10,70 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @RestController
 @CrossOrigin(origins = "http://localhost:3000")
 public class EnrollmentController {
 
     // instructor downloads student enrollments for a section, ordered by student name
     // user must be instructor for the section
+    @Autowired
+    private EnrollmentRepository enrollmentRepository;
+
+    @Autowired
+    private SectionRepository sectionRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private CourseRepository courseRepository;
+
+    @Autowired
+    private TermRepository termRepository;
+
     @GetMapping("/sections/{sectionNo}/enrollments")
     public List<EnrollmentDTO> getEnrollments(
             @PathVariable("sectionNo") int sectionNo ) {
 
         // TODO
 		//  hint: use enrollment repository findEnrollmentsBySectionNoOrderByStudentName method
         //  remove the following line when done
-        return null;
+
+        try {
```

Comment:
Your error handling is very clean. Going to have to implement try blocks. I used  _throw new_.  I think the try block adds more flexibility to handle errors better.